addon/services/tour.js

```diff
@@ -449,16 +449,19 @@ export default Service.extend(Evented, {
       const currentStep = tour.steps[index];
 
       currentStep.on('before-show', () => {
-        if (this.get('modal')) {
-          const currentElement = this.getElementForStep(currentStep);
-
-          if (currentElement) {
+        const currentElement = this.getElementForStep(currentStep);
+        if (currentElement) {
+          if (this.get('modal')) {
```

Comment:
I'm thinking we should remove this check here, and do `popoutElement` all the time, but do checks inside the `popoutElement` for if it's modal or not. That way, we only have one spot in the code that adds highlight classes. What do you think?